meson.build

```diff
@@ -291,7 +291,7 @@ if valgrind.found()
 endif
 if get_option('offline')
   if not get_option('systemd')
-    error('-Doffline=true requires -Dsystemd=true')
+    warning('-Doffline=true requires -Dsystemd=true')
```

Comment:
Although this fixes chrome CI isn't it more correct to have CI do the adjustment?

Reply:
Ya, it was just a bit antisocial to do this without some kind of a heads up. We could revert this after 1.7.2 drops.